appveyor.yml

```diff
@@ -11,28 +11,37 @@ environment:
 #        Build                    #
 #---------------------------------#
 
-build: off
-
-configuration: Release
-
 install:
   - SET PATH=%PYTHON%;%PYTHON%\\Scripts;%PATH%;C:\MinGW\bin
   - copy c:\MinGW\bin\mingw32-make.exe c:\MinGW\bin\make.exe
   - copy c:\Python36-x64\python.exe c:\Python36-x64\python3.exe
   - python --version
   - make install
-
-after_test:
   - make package
+  - copy /Y C:\projects\dexbot\dist\DEXBot-gui.exe C:\projects\dexbot\installer\windows\msi\DEXBot.exe
```

Comment:
Fix hard coded paths using `%APPVEYOR_BUILD_FOLDER%` (like below), and is this necessary at all?